src/services/submitChangePassword.ts

```diff
@@ -0,0 +1,39 @@
+'use server';
+import { api } from './api';
+
+export async function submitChangePassword({
+  password,
+  passwordConfirmation,
+}: {
+  password: string;
+  passwordConfirmation: string;
+}) {
+  try {
+    if (password !== passwordConfirmation) {
+      throw new Error('Senhas digitadas não coincidem');
+    }
```

Comment:
É uma boa prática fazer uma validação antes de fazer a requisição.
```suggestion
    if (password !== passwordConfirmation) {
      throw new Error('Senhas digitadas não coincidem');
    }
```